nf_core/pipeline-template/.github/workflows/cloud_tests_small.yml

```diff
@@ -0,0 +1,76 @@
+name: small-sized tests on cloud providers
+run-name: Submitting workflow to all cloud providers using small sized data
+on:
+  workflow_dispatch:
+    inputs:
+      platform:
+        description: "Platform to run test"
+        required: true
+        default: "all"
+        type: choice
+        options:
+          - all
+          - aws
+          - azure
+          - gcp
+jobs:
+  run-small-tests-on-aws:
+    if: ${{ github.event.inputs.platform == 'all' || github.event.inputs.platform == 'aws' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: seqeralabs/action-tower-launch@v2
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
+          compute_env: ${{ secrets.TOWER_CE_AWS_CPU }}
+          workdir: "${{ secrets.TOWER_BUCKET_AWS }}/work/{{ short_name }}/{% raw %}work-${{ github.sha }}"
+          run_name: "aws_fetchngs_small"
+          profiles: test
+          parameters: |
+            {
+                "outdir": "${{ secrets.TOWER_BUCKET_AWS }}/work/{{ short_name }}/{% raw %}results-test-${{ github.sha }}{% endraw %}"
+            }
+      - uses: actions/upload-artifact@v3
+        with:
+          name: Tower debug log file
+          path: tower_action_*.log
+  run-small-tests-on-gcp:
+    if: ${{ github.event.inputs.platform == 'all' || github.event.inputs.platform == 'gcp' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: seqeralabs/action-tower-launch@v2
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
+          compute_env: ${{ secrets.TOWER_CE_GCP_CPU }}
+          workdir: "${{ secrets.TOWER_BUCKET_GCP }}/work/{{ short_name }}/{% raw %}work-${{ github.sha }}"
+          run_name: "gcp_fetchngs_small"
+          profiles: test
+          parameters: |
+            {
+                "outdir": "${{ secrets.TOWER_BUCKET_GCP }}/work/{{ short_name }}/{% raw %}results-test-${{ github.sha }}{% endraw %}"
+            }
+      - uses: actions/upload-artifact@v3
+        with:
+          name: Tower debug log file
+          path: tower_action_*.log
+  run-small-tests-on-azure:
+    if: ${{ github.event.inputs.platform == 'all' || github.event.inputs.platform == 'azure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: seqeralabs/action-tower-launch@v2
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
+          compute_env: ${{ secrets.TOWER_CE_AZURE_CPU }}
+          workdir: "${{ secrets.TOWER_BUCKET_AZURE }}/work/{{ short_name }}/{% raw %}work-${{ github.sha }}"
```

Comment:
`{% endraw %}` missing at the end of the string

Reply:
Fixed now - thanks!